HeartRateMonitor.py

```diff
@@ -0,0 +1,21 @@
+class HRM:
+    avghr = -1
+    insthr = []
+    times_insthr = []
+    diagnosis = ''
+    times_diagnosis = []
+
+    def __init__(self, times=[], voltages=[], time_units='s'):
+        self.times = times
+        self.voltages = voltages
+
+    def avghr(self, t1=-1, t2=-1):
```

Comment:
Again, what does `-1` represent?  In this case, how are `t1` and `t2` not what they default to here?